src/chess_utils/game_etl.py

```diff
@@ -1,13 +1,16 @@
+from builtins import function
+
 import numpy as np
+from typing import List
 
 
-def array_reshape(data: np.array, n_point: int) -> np.ndarray:
+def array_reshape(data: np.ndarray, n_point: int) -> np.ndarray:
     """
-    Split an np array into np ndarray where each row has n_points.
+    Split an np array into np.ndarray where each row has n_points.
```

Comment:
Why the inconsistency between ```np array``` and ```np.ndarray```? And I think only ```np.ndarray``` is a class.

Reply:
Fixed.